client/server/middleware/assets.js

```diff
@@ -58,9 +58,11 @@ export default () => {
 				return EMPTY_ASSETS;
 			}
 
-			const allTheFiles = chunk.files.concat(
-				flatten( chunk.siblings.map( ( sibling ) => getChunkById( assets, sibling ).files ) )
-			);
+			const allTheFiles = chunk.chunks
```

Comment:
I assume the named chunks always have the `chunks` property derived from the `stats.namedChunkGroups`. But it seems to be safer to keep the old implementation.